www-plugins/passff-host/passff-host-1.0.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_4 )
+
+inherit python-single-r1
+
+DESCRIPTION="Host app for the PassFF WebExtension"
+HOMEPAGE="https://github.com/passff/passff-host"
+SRC_URI="
+	https://github.com/passff/passff-host/releases/download/${PV}/passff.py   -> ${P}.py
+	https://github.com/passff/passff-host/releases/download/${PV}/passff.json -> ${P}.json
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="chrome chromium firefox vivaldi"
+
+S="${WORKDIR}"
+
+src_unpack() {
+	cp "${DISTDIR}/${P}.json" passff.json || die
+	cp "${DISTDIR}/${P}.py" passff.py || die
+	python_fix_shebang passff.py
+}
+
+src_install() {
+	for browser in $IUSE; do
+		case "$browser" in
+			firefox)  TARGET_DIR="/usr/lib/mozilla/native-messaging-hosts" ;;
+			chrome)   TARGET_DIR="/etc/opt/chrome/native-messaging-hosts" ;;
+			chromium) TARGET_DIR="/etc/chromium/native-messaging-hosts" ;;
+			vivaldi)  TARGET_DIR="/etc/vivaldi/native-messaging-hosts" ;;
+		esac
+
+		sed "s;PLACEHOLDER;${TARGET_DIR};g" passff.json > "passff-${browser}.json" || die
```

Comment:
I suppose you can `doins` it first, then `sed` inside the install location to avoid renaming it back and forth.

Reply:
sed into the install location gives:
```
 * ACCESS DENIED:  mkostemp:     /etc/chromium/native-messaging-hosts/sedXXXXXX
sed: couldn't open temporary file /etc/chromium/native-messaging-hosts/sedXXXXXX: Permission denied
```

Reply:
You need to sed relatively to `${ED}`.